WORKING_GROUPS.md

```diff
@@ -419,7 +419,7 @@ Responsibilities include:
 * Offer help to npm package maintainers to fix high-impact security bugs.
 * Maintain and make available data on disclosed security vulnerabilities in:
   * the core Node.js project
-  * other projects maintained by the Node.js Foundation technical group
+  * other projects maintained by the OpenJS Foundation technical group
```

Comment:
I don't know if there is such an "OpenJS Foundation technical group"

Reply:
Looking at the list I think we should just delete everything up to the line "Promote the improvement of security practices within the Node.js ecosystem." as the vulnerability reporting part was dropped a little while back.

Reply:
We should probably re-charter the group with that removed too.

Reply:
Do you want to create a new issue before resolving this conversation thread @Trott ?

Reply:
I created a new issue for you so I can resolve this comment.
https://github.com/nodejs/TSC/issues/1081